app/data/test_skip_condition_group.json

```diff
@@ -0,0 +1,118 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "questionnaire_id": "0",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.1",
+    "survey_id": "0",
+    "title": "Skip group",
+    "theme": "default",
+    "description": "A questionnaire to skipping a group.",
+    "eq_id": "0",
+    "messages": {
+      "INTEGER_TOO_LARGE": "Number is too large",
```

Comment:
Do we actually support these at the top-level like this, would have saved copy/pasting them against each answer in the Census schema!